keystorage/aws/src/main/java/tech/pegasys/signers/aws/CacheableAwsSecretsManagerProvider.java

```diff
@@ -17,20 +17,24 @@
 
 import com.google.common.cache.Cache;
 import com.google.common.cache.CacheBuilder;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider;
+import software.amazon.awssdk.regions.providers.DefaultAwsRegionProviderChain;
 
 public class CacheableAwsSecretsManagerProvider {
```

Comment:
Wondering now is the time to merge CacheableAwsSecretsManagerProvider into AwsSecretsManager as we previously discussed?

I don't think there's a reason to have the non-cacheable version (if we _do_ want to provide a choice of two, then I think the API should be normalised rather than having two patterns for constructing AwsSecretsManager).

Reply:
Agree, there is no need to have a non-cacheable version.